zuul.d/molecule.yaml

```diff
@@ -1,305 +1,304 @@
 - job:
-    name: cifmw-molecule-repo_setup
+    files:
+    - ^ansible-requirements.txt
+    - ^molecule-requirements.txt
+    - ^ci_framework/roles/test_deps/(?!meta|README).*
+    - ^ci/playbooks/molecule.*
+    name: cifmw-molecule-test_deps
     parent: cifmw-molecule-base
     vars:
-      TEST_RUN: repo_setup
-    files:
-      - ^ansible-requirements.txt
-      - ^molecule-requirements.txt
-      - ^ci_framework/roles/repo_setup/(?!meta|README).*
-      - ^ci/playbooks/molecule.*
+      TEST_RUN: test_deps
 - job:
-    name: cifmw-molecule-rhol_crc
-    nodeset: centos-9-crc-xl
+    files:
+    - ^ansible-requirements.txt
+    - ^molecule-requirements.txt
+    - ^ci_framework/roles/ci_setup/(?!meta|README).*
+    - ^ci/playbooks/molecule.*
+    name: cifmw-molecule-ci_setup
     parent: cifmw-molecule-base
     vars:
-      TEST_RUN: rhol_crc
-    files:
-      - ^ansible-requirements.txt
-      - ^molecule-requirements.txt
-      - ^ci_framework/roles/rhol_crc/(?!meta|README).*
-      - ^ci/playbooks/molecule.*
+      TEST_RUN: ci_setup
 - job:
-    name: cifmw-molecule-run_hook
+    files:
+    - ^ansible-requirements.txt
+    - ^molecule-requirements.txt
+    - ^ci_framework/roles/edpm_deploy/(?!meta|README).*
+    - ^ci/playbooks/molecule.*
+    name: cifmw-molecule-edpm_deploy
     parent: cifmw-molecule-base
     vars:
-      TEST_RUN: run_hook
-    files:
-      - ^ansible-requirements.txt
-      - ^molecule-requirements.txt
-      - ^ci_framework/roles/run_hook/(?!meta|README).*
-      - ^ci/playbooks/molecule.*
+      TEST_RUN: edpm_deploy
 - job:
-    name: cifmw-molecule-set_openstack_containers
-    parent: cifmw-molecule-base-crc
-    vars:
-      TEST_RUN: set_openstack_containers
     files:
-      - ^ansible-requirements.txt
-      - ^molecule-requirements.txt
-      - ^ci_framework/roles/set_openstack_containers/(?!meta|README).*
-      - ^ci/playbooks/molecule.*
-- job:
-    name: cifmw-molecule-tempest
+    - ^ansible-requirements.txt
+    - ^molecule-requirements.txt
+    - ^ci_framework/roles/hive/(?!meta|README).*
+    - ^ci/playbooks/molecule.*
+    name: cifmw-molecule-hive
     parent: cifmw-molecule-base
     vars:
-      TEST_RUN: tempest
+      TEST_RUN: hive
+- job:
     files:
-      - ^ansible-requirements.txt
-      - ^molecule-requirements.txt
-      - ^ci_framework/roles/tempest/(?!meta|README).*
-      - ^ci/playbooks/molecule.*
+    - ^ansible-requirements.txt
+    - ^molecule-requirements.txt
+    - ^ci_framework/roles/build_openstack_packages/(?!meta|README).*
+    - ^ci/playbooks/molecule.*
+    name: cifmw-molecule-build_openstack_packages
+    parent: cifmw-molecule-base
+    vars:
+      TEST_RUN: build_openstack_packages
 - job:
-    name: cifmw-molecule-test_deps
+    files:
+    - ^ansible-requirements.txt
+    - ^molecule-requirements.txt
+    - ^ci_framework/roles/operator_deploy/(?!meta|README).*
+    - ^ci/playbooks/molecule.*
+    name: cifmw-molecule-operator_deploy
```

Comment:
it seems to lack the right nodeset - did you ensure you imported the current state?

Reply:
Added nodeset https://github.com/openstack-k8s-operators/ci-framework/pull/401/files#diff-e11625dd628d0cfeeaad59207c1fabdd877fb86c87eeca532b83571e184628dcR15